src/tests/ftest/control/dmg_pool_query_test.yaml

```diff
@@ -35,7 +35,7 @@ exp_vals:
   replicas: [0]
   query_mask: "rebuild,space"
   scm:
-    total: 16000008192
+    total: 16000000000
```

Comment:
Is this test this picky or are you just making this a clean number?

Reply:
This is now being specified as base-10 not base-2, I don't know whether that's a result of this PR or an existing failure on the branch. I don't have anything to compare with that is running features pool and control. I expect that it's a result of now reading the SCM quantity from meta-blob size which I imagine is specified in base-10 as its storage rather than the VOS-file size in RAM/memory which is specified in base-2.

Reply:
Because pool create uses `scm_size: 16GB`, this looks correct to me. Thanks for the explanation